themes/components-training-theme/src/css/variables/_buttons.scss

```diff
@@ -0,0 +1,65 @@
+$btn-primary: () !default;
+$btn-primary: map-deep-merge(
+	(
+		background-color: $primary,
+		border-color: $primary,
+		color: $white,
+        border-radius: 0,
```

Comment:
instead of overriding the `border-radius` within the map functions, we could use the Sass variables, which would be much simpler to add back if we needed to in the future. Other than that, it all looks good to me.